.github/workflows/publish-branch.yaml

```diff
@@ -0,0 +1,77 @@
+name: Publish Branch
+
+on:
+    push:
+        branches:
+            - main
+            - dev
+
+jobs:
+    release_candidate:
+        runs-on: ubuntu-latest
+        env:
+            NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+            NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+            BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
+        steps:
+            - uses: actions/checkout@v4
+              with:
+                  ref: ${{ github.head_ref }}
+            - uses: actions/setup-node@v4
+              with:
+                  node-version: "16.x"
```

Comment:
Can we do 18? Or better yet use the shared ./.github/actions/setup

Reply:
Not tested, but applied